src/aura/BGE_EntryForm/BGE_EntryForm.cmp

```diff
@@ -62,27 +67,38 @@
             </lightning:layoutItem>
 
             <lightning:layoutItem size="12" smallDeviceSize="6" mediumDeviceSize="4" largeDeviceSize="2" class="slds-p-horizontal_small">
-                <aura:if isTrue="{!v.donorType == 'Contact1'}">
-                    <label for="contactLookup" class="slds-form-element__label">{!v.labels.contactObject}</label>
+                <!--don't use aura:if to hide, because we need to get reference to the inputfield for data validation-->
+                <div class="{!if(v.donorType == 'Account1', 'slds-hide', '')}">
```

Comment:
If we're just hiding, it is possible for a user to select Contact as Donor Type, select a contact record, change Donor Type to Account, select an account, and have both populated? Or is that handled somewhere?

Reply:
Good catch here too! I knew we'd been using aura:ifs for reasons... will put something in place for this.